pkg/plugins/policies/meshretry/api/v1alpha1/schema.yaml

```diff
@@ -0,0 +1,171 @@
+    properties:
+      targetRef:
+        description: TargetRef is a reference to the resource the policy takes an effect on. The resource could be either a real store object or virtual resource defined inplace.
+        properties:
+          kind:
+            description: Kind of the referenced resource
+            enum:
+              - Mesh
+              - MeshSubset
+              - MeshService
+              - MeshServiceSubset
+              - MeshGatewayRoute
+            type: string
+          mesh:
+            description: Mesh is reserved for future use to identify cross mesh resources.
+            type: string
+          name:
+            description: 'Name of the referenced resource. Can only be used with kinds: `MeshService`, `MeshServiceSubset` and `MeshGatewayRoute`'
+            type: string
+          tags:
+            additionalProperties:
+              type: string
+            description: Tags used to select a subset of proxies by tags. Can only be used with kinds `MeshSubset` and `MeshServiceSubset`
+            type: object
+        type: object
+      to:
+        description: To list makes a match between the consumed services and corresponding configurations
+        items:
+          properties:
+            default:
+              description: Default is a configuration specific to the group of destinations referenced in 'targetRef'
+              properties:
+                grpc:
+                  description: GRPC defines a configuration of retries for GRPC traffic
+                  properties:
+                    backOff:
+                      description: BackOff is a configuration of durations which will be used in exponential backoff strategy between retries
+                      properties:
+                        baseInterval:
+                          description: BaseInterval is an amount of time which should be taken between retries. Must be greater than zero. Values less than 1 ms are rounded up to 1 ms. Default is 25ms.
+                          type: string
+                        maxInterval:
+                          description: MaxInterval is a maximal amount of time which will be taken between retries. Default is 10 times the "BaseInterval".
+                          type: string
+                      type: object
+                    numRetries:
+                      description: NumRetries is the number of attempts that will be made on failed (and retriable) requests
+                      format: int32
+                      type: integer
+                    perTryTimeout:
+                      description: PerTryTimeout is the amount of time after which retry attempt should timeout. Setting this timeout to 0 will disable it. Default is 15s.
+                      type: string
+                    retryOn:
+                      description: RetryOn is a list of conditions which will cause a retry.
```

Comment:
should we specify available values here?

Reply:
I don't know, it's not enum since you can put any status code here. But definetely I can improve the `description`

Reply:
It's not an enum, but we validate values. We know the set that we accept therefore it would be useful to present this set.